frontend/pages/hosts/HostDetailsPage/HostDetailsPage.jsx

```diff
@@ -0,0 +1,336 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+
+import ReactTooltip from 'react-tooltip';
+import { noop, pick } from 'lodash';
+
+import Spinner from 'components/loaders/Spinner';
+import Button from 'components/buttons/Button';
+import Modal from 'components/modals/Modal';
+
+import entityGetter from 'redux/utilities/entityGetter';
+import { renderFlash } from 'redux/nodes/notifications/actions';
+import { push } from 'react-router-redux';
+import PATHS from 'router/paths';
+
+import hostInterface from 'interfaces/host';
+import { humanMemory, humanUptime, humanLastSeen, humanEnrolled } from 'components/hosts/HostsTable/helpers';
```

Comment:
I feel a little uneasy that helpers are being imported from a different component. Maybe these helpers can be added to `frontend/kolide/helpers.js` and imported from there to both components?